.github/workflows/cd.yml

```diff
@@ -20,10 +20,17 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: 1.8
+    - name: download heroku cli
+      uses: wei/curl@master
+      with:
+        args: https://cli-assets.heroku.com/install.sh >> ./.github/heroku_install.sh
```

Comment:
Why you need to put it in `./.github` folder under a different name?

Reply:
2f299c3